pages/app/presenters/refinery/pages/section_presenter.rb

```diff
@@ -85,5 +83,23 @@ def sanitize_content(input)
         return output
       end
     end
+
+    class CustomScrubber < Rails::Html::PermitScrubber
+      def initialize(tags, attributes)
+        @direction = :bottom_up
+        @tags = tags
+        @attributes = attributes
+      end
+
+      #see https://github.com/flavorjones/loofah/blob/master/lib/loofah/html5/scrub.rb#L21
+      def scrub_attributes(node)
+        node.attribute_nodes.each do |attr_node|
+          next if attr_node.node_name =~ /\Adata-[\w-]+\z/
```

Comment:
`/\Adata-[\w-]+\z/` Why do we hardcoded this regex and don't let the dev to be able to override it in a config?